tutorials/preprocessing/plot_50_artifact_correction_ssp.py

```diff
@@ -402,6 +403,40 @@
 # projectors are capturing something else other than a heartbeat artifact (and
 # thus may be removing brain signal and should be discarded).
 #
+# .. _tut-artifact-ssp-reconstruction:
+#
+# Reducing SSP bias in sensor space via signal reconstruction
+# ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+# In addition to looking at unprojected versus projected data, it's also useful
+# to look at sensor space data reconstructed to reduce the bias introduced by
```

Comment:
I think this is the first mention in the tutorial of "bias introduced by projection" and thus I think it needs explaining. Can you add a new sentence before this one, explaining that projection introduces bias, what kind of bias it is, how/why it happens, when it might be a problem for your analysis, etc?

Reply:
[Better](https://21289-1301584-gh.circle-artifacts.com/0/dev/auto_tutorials/preprocessing/plot_50_artifact_correction_ssp.html#visualizing-ssp-sensor-space-bias-via-signal-reconstruction) ?

Reply:
yep. thanks!